test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/TagHelperDescriptorResolverTest.cs

```diff
@@ -689,24 +689,38 @@ public void DescriptorResolver_DoesNotReturnInvalidTagHelpersWhenSpecified()
             Assert.Empty(descriptors);
         }
 
+        public static TheoryData<string> DescriptorResolver_IgnoresSpacesData
```

Comment:
I totally missed updating tests when I first sent the PR. The `TagHelperDescriptorFactoryTests` changes are pretty trivial. This one is a bit more interesting - in the old code, we were flowing `Microsoft.AspNet.Razor.Runtime.TagHelpers.Valid_PlainTagHelper,MyAssembly` as the lookup text but resolving the types through the current assembly.
Consequently, the descriptor had an assembly name different than the lookup text and the test at https://github.com/aspnet/Razor/blob/dev/src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperDescriptorResolver.cs#L122 would fail resulting in an empty descriptor result.

With the change I made, the descriptor uses the passed in assembly name. Consequently, the check would pass and we'd have a TagHelperDescriptor that matched the passed in lookup text (in terms of type name + assembly). This is essentially how we make precompilation work - we produce a secondary assembly but have it behave as if the assembly name matched the original assembly.

If this was intentionally designed to behave this way, I can change this.